ap_api/resources/patch-transactions.api.mdx

```diff
@@ -0,0 +1,104 @@
+---
+id: patch-transactions
+title: "Accepts updated information on transactions."
+description: "Accepts one or more objects containing updated information on transactions. Note that requests containing invalid data for any transaction will result in a 400 Bad Request and no transactions will be updated. Note that this endpoint accepts a subset of transaction information defined in the `PatchTransaction` schema."
```

Comment:
We may want to clarify that currently, requests to this endpoint are not atomic. Updates will be applied one-by-one until the first error occurs, if any.

Reply:
It's written in the reference guide. I can copy that section here